packages/react/src/patterns/blocks/ContentBlockMedia/README.md

```diff
@@ -33,9 +33,9 @@ function App() {
       title: 'Aliquam condimentum interdum',
       image: {
         uri: {
-          sm: 'https://via.placeholder.com/640x320',
-          md: 'https://via.placeholder.com/768x384',
-          lg: 'https://via.placeholder.com/1024x512',
+          sm: 'https://dummyimage.com/640x320',
+          md: 'https://dummyimage.com/768x384',
+          lg: 'https://dummyimage.com/1024x512',
```

Comment:
Should these have an aspect ratio and new styles?

Reply:
I didn't bother to put that in this part of the README, as I'd have to document the service's API. If you think it should be in the docs we can add a few lines for usage guidance.